src/translations/en.json

```
@@ -4389,7 +4389,7 @@
         "log": "Log"
       },
       "configuration": {
-        "no_configuration": "This add-on does not expose configuration for you to mess with…",
+        "no_configuration": "This add-on does not have configurations…",
```

I think this sounds a bit more natural:

```suggestion
        "no_configuration": "This add-on has no configuration…",
```